examples/repl_examples.ergo

```diff
@@ -24,3 +24,9 @@ enforce 5 = match 1 with 2 then 5 else 5; return "(match) defaults work"; // def
 let sauce = "hot" ; enforce "sriracha" = match sauce with "mild" then "ketchup" with "hot" then "sriracha" else nil ; return "(match) multiple clauses work"; // multiple clauses (spicy)
 
 enforce 1.0 = match 1.0 with let a then a else 3.0; return "(match) plain old let works"; // let
+return flatten([[]]);
```

Comment:
usually I use an enforce here so that it's obvious in the output if anything failed. e.g. if `max` starts returning the min instead of the max (which is a totally possible bug), how would I know looking at the output?